terraform/groups/identity-gateway/main.tf

```diff
@@ -22,6 +22,7 @@ module "ecs" {
   source       = "./modules/ecs"
   service_name = var.service_name
   vpc_id       = data.aws_vpc.vpc.id
+  tags         = local.common_tags
```

Comment:
In `locals.tf` this is defined as `locals` rather than `local`

Reply:
That's a fun thing terraform does just because. `locals` when defined `local` when referenced